## Задание 1:
Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку.   
Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%.   
Дайте краткие рекомендации команде.   
Результаты:  
* 1) Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550.   
* 2) Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228   
* 3) Конверсии : conv1 = 1.05% conv2 = 1.47%

In [1]:
import numpy as np
import statsmodels.stats.proportion as smp

In [2]:
alpha = 0.05
# число успехов в каждой группе
successes = np.array([164, 228])

# число наблюдений в каждой группе
nobs = np.array([15550, 15550])

# пропорции успехов в каждой группе
proportions = successes / nobs

# z-статистика и p-значение
zstat, pvalue = smp.proportions_ztest(count=successes, nobs=nobs, value=0, alternative='two-sided')
print(zstat)
print(pvalue)

-3.253054722696254
0.001141715077067056


На основе z-статистики: -3.253054722696254 и p-value: 0.001141715077067056, можно сделать вывод о статистически значимых различиях между конверсиями в двух группах.  

    Рекомендации команде: на основании результатов эксперимента можно рекомендовать использовать упрощенную форму заказа в магазине Утконос, так как она приводит к более высокой конверсии в покупку. Однако, следует провести дополнительные исследования, чтобы понять, какие именно элементы формы заказа необходимо упростить, чтобы максимизировать конверсии.